src/core/version_neg.h

```diff
@@ -65,21 +71,13 @@ QuicVersionNegotiationExtGenerateCompatibleVersionsList(
     );
 
 QUIC_STATUS
-QuicVersionNegotiationExtParseClientVerNegInfo(
-    _In_ QUIC_CONNECTION* Connection,
-    _In_reads_bytes_(BufferLength)
-        const uint8_t* const Buffer,
-    _In_ uint16_t BufferLength,
-    _Out_ QUIC_CLIENT_VER_NEG_INFO* ClientVNI
-    );
-
-QUIC_STATUS
-QuicVersionNegotiationExtParseServerVerNegInfo(
+QuicVersionNegotiationExtParseVersionInfo(
     _In_ QUIC_CONNECTION* Connection,
     _In_reads_bytes_(BufferLength)
         const uint8_t* const Buffer,
     _In_ uint16_t BufferLength,
-    _Out_ QUIC_SERVER_VER_NEG_INFO* ServerVNI
+    _In_ BOOLEAN IsServer,
```

Comment:
`IsServer` should be able to be inferred from `Connection`, right?

Reply:
The Connection is only used for logging, and I'm a little hesitant to actually use the pointer because it'll break the tests that don't use a real connection object.